### PR TITLE
Button Plugins + Moved EditHTML button to custom buttons

### PR DESCRIPTION
**Pull request by stevermeister**

### CODE REVIEW

**Comment by stevermeister** [comment]
@bampakoa what do you think?

File: projects/ngx-wig/src/lib/config.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export type TButton = {
   command?: string;
   styleClass?: string;
   pluginName?: string;
-  isComplex?: boolean;
+  isPlugin?: boolean;
```
Comment (bampakoa) [inline projects/ngx-wig/src/lib/config.ts]: Would it worth to get rid of the `boolean` property and check the existence of a plugin with `pluginName` property?

File: projects/ngx-wig/src/lib/config.ts [hunk 2]
```diff
@@ -59,4 +62,18 @@ export const DEFAULT_LIBRARY_BUTTONS: TButtonLibrary = {
   }
 };
 
+export const CUSTOM_LIBRARY_BUTTONS: TButtonLibrary = {
```
Comment (bampakoa) [inline projects/ngx-wig/src/lib/config.ts]: I really like the simplicity of this approach. Definitely much cleaner than previous suggestions. I have some concerns though.

This approach seems a good fit for button plugins. So, I would suggest to make it clear to the consumers of the library that they can create **only** button plugins and not generally plugins (that would involve interaction with their components and we would need, possibly, more than a simple command handler) 😉

File: projects/ngx-wig/src/lib/config.ts (hunk 1 above)
Comment (stevermeister) [reply projects/ngx-wig/src/lib/config.ts]: true, I even don't know whether we need second one.
I'll cleanup both

File: projects/ngx-wig/src/lib/config.ts (hunk 2 above)
Comment (stevermeister) [reply projects/ngx-wig/src/lib/config.ts]: I think with buttons we can cover 9 of 10 cases.

Comment (bampakoa) [reply projects/ngx-wig/src/lib/config.ts]: Should we name it `ButtonPlugin` or `CustomButton` or something that denotes button so that it is more clear?

**Comment by bampakoa** [comment]
@stevermeister LGTM!

**Comment by stevermeister** [comment]
published 2.1.0 - https://www.npmjs.com/package/ngx-wig

btw, @bampakoa do you know whether we need `tslib`?

**Comment by bampakoa** [comment]
@stevermeister because of `importHelpers` https://github.com/stevermeister/ngx-wig/blob/master/tsconfig.json#L12

**Pull request merged**